src/__tests__/iris.ts

```diff
@@ -44,6 +44,7 @@ describe('iris dataset test method covarianceMatrix', () => {
 
 describe('iris dataset test wrong method', () => {
   it('wrong method', () => {
+    // @ts-ignore
```

Comment:
```suggestion
    // @ts-expect-error
```

same for the other tests